django-stubs/utils/choices.pyi

```diff
@@ -0,0 +1,31 @@
+from collections.abc import Iterable, Iterator
+from typing import Any, Protocol, TypeVar, type_check_only
+
+from typing_extensions import TypeAlias
+
+_Choice: TypeAlias = tuple[Any, Any]
+_ChoiceNamedGroup: TypeAlias = tuple[str, Iterable[_Choice]]
+_Choices: TypeAlias = Iterable[_Choice | _ChoiceNamedGroup]
```

Comment:
Is the any reason to change the name from `_FieldChoices` to `_Choices`? I think that keeping the old name can safe a lot of changed lines.

Reply:
I wanted to indicate that `_FieldChoices` is no longer just a type for `Field`s, as it has been moved from `django.db.models.fields` to `django.utils.choices`.

But I'm not sure if this approach is correct 😂

Reply:
I am fine with both, just double checking.